src/backend/base/langflow/custom/custom_component/base_component.py

```diff
@@ -35,12 +35,12 @@ def __init__(self, **data):
         self.cache = TTLCache(maxsize=1024, ttl=60)
         for key, value in data.items():
             if key == "user_id":
-                setattr(self, "_user_id", value)
+                self._user_id = value
             else:
                 setattr(self, key, value)
 
     def __setattr__(self, key, value):
-        if key == "_user_id" and hasattr(self, "_user_id") and getattr(self, "_user_id") is not None:
+        if key == "_user_id" and self._user_id is not None:
```

Comment:
`_user_id` is a class attribute so it seems it will always be there ?

Reply:
Agree.